packages/dashboard-message-bus/lib/message/types.ts

```diff
@@ -41,6 +41,16 @@ export interface InvalidateMessage extends Message {
   payload: number;
 }
 
+/**
+ * Message intended to set analytic collection preference for user.
+ * The payload indicate whether or not the user opts in to share anonymous usage data with Truffle.
+ * This message payload is a truffle command request to update user config that should be forwarded.
+ */
+export interface SetAnalytics extends Message {
```

Comment:
Looks like the other message types have `Message` in the name:

```suggestion
export interface SetAnalyticsMessage extends Message {
```

Reply:
(You'll have to update the other part of the code, also)

Reply:
thank you @gnidan 
I will update it.